src/util/redux.spec.js

```diff
@@ -0,0 +1,197 @@
+import assert from 'assert';
+import sinon from 'sinon';
+import {
+	assign,
+	isFunction,
+	upperCase
+} from 'lodash';
+
+import {
+	thunk,
+	getReduxPrimitives
+} from './redux';
+
```

Comment:
also they all should be grouped in a top-level describe that labels the whole module. E.g. `describe('redux utils')`